src/python_routing_v02/compute_nhd_routing_SingleSeg.py

```diff
@@ -413,11 +423,11 @@ def singlesegment(
 def main():
 
     args = _handle_args()
-
+    global dt 
```

Comment:
Probably better as a local scope declaration.

Reply:
the main() function is not tied up with everything that runs in the first part of the file. i set it to global so that it would be recognized for the fortran section. I can wrap everything inside the main function.